deployment/build.portable

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#= Portable .NET DLL .tar.gz
```

Comment:
wtf? :D

Reply:
It's a comment artifact from a copy. I don't care.